app/containers/Itunes Provider/TrackDetailsContainer/index.js

```diff
@@ -0,0 +1,164 @@
+import React, { memo, useState, useEffect } from 'react';
+import { compose } from 'redux';
+import PropTypes from 'prop-types';
+import { Skeleton, Card } from 'antd';
+import { useParams } from 'react-router-dom';
+import { get, isEmpty } from 'lodash';
+import styled from 'styled-components';
+import { connect } from 'react-redux';
+import { injectSaga } from 'redux-injectors';
+import { injectIntl } from 'react-intl';
+import { createStructuredSelector } from 'reselect';
+import { selectTrackErrors, selectSearchedTrack } from '../selectors';
+import If from '@components/If';
+import T from '@components/T';
+import For from '@components/For';
+import { searchContainerCreators } from '../reducer';
+import TrackDetailsComponent from '@app/components/TrackDetailsComponent/index';
+import searchContainerSaga from '../saga';
+
+const CustomCard = styled(Card)`
+  && {
+    margin: 20px 0;
+    max-width: 1000;
+    border-style: hidden;
+    background-color: #231f20;
+    height: 100vh;
```

Comment:
this becomes a problem when there is overflow. Can we set min-height at 100vh instead?